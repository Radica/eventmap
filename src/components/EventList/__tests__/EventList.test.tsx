// import React from 'react';
// import { render } from '@testing-library/react';
// import { MemoryRouter } from 'react-router-dom';

// import EventList from '../index';

describe('<EventList />', () => {
    it('renders', () => {
        // const mockData = [{ id: '1', name: 'Welly' }];
        // const tree = render(
        //     <MemoryRouter>
        //         <EventList list={mockData} />
        //     </MemoryRouter>
        // ).container.firstChild;
        //
        // expect(tree).toMatchSnapshot();
    });
});
