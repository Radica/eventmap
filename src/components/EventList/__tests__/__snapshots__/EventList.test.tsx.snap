// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventList /> renders 1`] = `
<div
  class="EventList"
>
  <h4>
    User List
  </h4>
  <ul>
    <li>
      <a
        href="/UserInfo/1"
      >
        Welly
      </a>
    </li>
  </ul>
</div>
`;
